team-meeting-notes/2021-02-18.md

```diff
@@ -0,0 +1,59 @@
+
+# Package team meeting 18th Feb 2021
+
+Attendees: 
+	Jesper, Kevin, Lottie, Nik, Andy, David
+
+Notetaker:  Kevin
+
+Agenda
+ - Status on Umbpack
+ - Status on Package Templates
+ - Status on UI Examples
+ - Status on Our
+ - Any other business
+
+## Status on Umbpack
+ - Outstanding PR (by jesper)- requires update to our (we need to confirm how/when this will be done).
```

Comment:
```suggestion
 - [Outstanding PR](https://github.com/umbraco/UmbPack/pull/59) (by jesper)- requires update to our (we need to confirm how/when this will be done).
```